### PR TITLE
added favorite cities for forecasts using redux

### PR DESCRIPTION
**Pull request by GedasIncirauskas**
1. Created redux store for save and delete favorite cities regarding forecasts.
2. Added responsive style for mobiles.

Screenshot 2022-05-17 131227
Screenshot 2022-05-17 150844

### CODE REVIEW

File: src/pages/cityWeather/CityWeather.jsx [hunk 1]
```diff
@@ -24,38 +28,73 @@ const CityWeather = () => {
...
+    setChooseCityForecast([]);
+  };
+
+  const findFavoriteCity = city => {
+    getCurrentCity(city);
+  };
+
+  const isCityExist = cityTitle => {
+    const isExist = searchResults.some(({ name }) => name === cityTitle);
+    return isExist;
```
Comment (JuliaShavyrova) [inline src/pages/cityWeather/CityWeather.jsx]: function can be shortened

File: src/pages/cityWeather/CityWeather.jsx [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 
 const CityWeather = () => {
   const [cityName, setCityName] = useState('');
-  const [selectedCity, setSelectedCity] = useState('');
+  const [selectedCity, setSelectedCity] = useState({});
   const [results, setResults] = useState([]);
   const [chooseCityForecast, setChooseCityForecast] = useState([]);
 
+  const dispatch = useDispatch();
+  const searchResults = useSelector(state => state.savedSearch);
```
Comment (JuliaShavyrova) [inline src/pages/cityWeather/CityWeather.jsx]: please think about naming for variables
it's not clear what's the difference between `results` and `searchResults`, and `cityName` actually is not a city name, but input value to search

File: src/config/constants.js [hunk 3]
```diff
@@ -3,3 +3,7 @@ export const MAIN_URL = 'https://pfa.foreca.com/api/v1';
 export const minSearchCharacters = 3;
 export const EMAIL_REGEXP = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
 export const minStarLength = [...Array.from({ length: 5 }, (_, i) => i + 1)];
+
+// Reducer types
+export const SAVE_SEARCH = 'SAVE_SEARCH';
```
Comment (JuliaShavyrova) [inline src/config/constants.js]: please move action names to more specific file or rename so it would be clear what way the variables should be used

Comment (GedasIncirauskas) [reply src/config/constants.js]: I created new file `actionTypes.js` and moved variables here.

File: src/pages/cityWeather/CityWeather.jsx (hunk 1 above)
Comment (GedasIncirauskas) [reply src/pages/cityWeather/CityWeather.jsx]: Fixed :)

File: src/pages/cityWeather/CityWeather.jsx (hunk 2 above)
Comment (GedasIncirauskas) [reply src/pages/cityWeather/CityWeather.jsx]: It was really hard to give a new names for variables, but hope I changed correctly.

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**